modules/variants/src/main/frontend/src/variantFilesContainer.jsx

```diff
@@ -660,118 +696,181 @@ export default function VariantFilesContainer() {
 
   return (
   <React.Fragment>
-    <form method="POST"
-          encType="multipart/form-data"
-          onSubmit={upload}
-          key="file-upload">
-      <Typography component="h2" variant="h5" className={classes.dialogTitle}>Variants Upload</Typography>
-      { uploadInProgress && (
-        <Grid item className={classes.root}>
-          <LinearProgress color="primary" />
+    <Typography variant="h2">Variants Upload</Typography>
+      <form method="POST"
+            encType="multipart/form-data"
+            onSubmit={upload}
+            key="file-upload"
+            id="variantForm">
+        <Grid container direction="row-reverse" justify="flex-end">
+          <Grid item md={12} lg={6}>
+            <Alert severity="info">
+              <AlertTitle>Expected file name format</AlertTitle>
+              <div>PatientId_TumorNumber.csv (e.g. AB12345_1.csv)</div>
+              <div>PatientId_TumorNumber_TumorRegion.csv (e.g. AB12345_1_a.csv)</div>
```

Comment:
Per @sashaandjic 's suggestion:
```suggestion
              <div>Patient_Tumor.csv (e.g. AB12345_1.csv)</div>
              <div>Patient_Tumor_TumorRegion.csv (e.g. AB12345_1_a.csv)</div>
```

Reply:
Done in 466ffc8.